webrtc.html

```diff
@@ -1521,9 +1521,10 @@ <h4>Set the RTCSessionDescription</h4>
                           </li>
                           <li>
                             <p>Fire an event named <code><a data-link-for=
-                            "RTCDataChannel">error</a></code > with an
-                            <code>OperationError</code> exception at
-                            <var>channel</var>.</p>
+                            "RTCDataChannel">error</a></code > using the
+                            <code><a>RTCErrorEvent</a></code> interface with its
+                            <code>errorDetail</code> attribute set to
+                            "data-channel-failure" at <var>channel</var>.</p>
```

Comment:
When I search for "data-channel-failure", I find two places that look inconsistent:

*"Fire an event named error using the RTCErrorEvent interface with its errorDetail attribute set to "data-channel-failure" at channel."*

vs. [here](https://pr-preview.s3.amazonaws.com/stefhak/webrtc-pc/pull/1971.html#dfn-data-channel-failure):

*"Fire an RTCError event at channel with errorDetail set to "data-channel-failure"."*

We should probably make it consistent.

Reply:
I _think_ that is already part of #1966